src/main/java/org/tdl/vireo/controller/SubmissionController.java

```diff
@@ -990,9 +990,18 @@ public ApiResponse removeFile(@WeaverUser User user, @PathVariable Long submissi
             apiResponse = new ApiResponse(ERROR, "You are not allowed to delete license files!");
         } else {
             if (user.getRole().equals(Role.ROLE_ADMIN) || user.getRole().equals(Role.ROLE_MANAGER) || uri.contains(String.valueOf(hash))) {
-                JsonNode fileInfo = assetService.getAssetFileInfo(uri, submission);
+                String fileName = "";
+                String fileSize = "file not found";
+                if (assetService.assetFileExists(uri)) {
```

Comment:
Reasonable method but why condition. Shouldn't application know the file exists already?